command/format/diff.go

```diff
@@ -359,9 +350,6 @@ func (p *blockBodyDiffPrinter) writeAttrDiff(name string, attrS *configschema.At
 
 	if attrS.Sensitive {
 		p.buf.WriteString("(sensitive value)")
-		if p.pathForcesNewResource(path) {
-			p.buf.WriteString(p.color.Color(forcesNewResourceCaption))
-		}
```

Comment:
I think there's a bad rebase here: this is undoing the fix in #28583, and further down we also replace the test case.